cmd/vpn_server/app/firewall.go

```diff
@@ -35,19 +37,20 @@ func firewallCommand() *cobra.Command {
 			if err != nil {
 				return err
 			}
-			return runFirewallCommand(log, device, mode, shootNetworks)
+			return runFirewallCommand(log, device, mode, shootNetworks, seedPodNetwork)
 		},
 	}
 
 	cmd.Flags().StringVar(&device, "device", "", "device to configure")
 	cmd.Flags().StringVar(&mode, "mode", "", "mode of firewall (up or down)")
 	cmd.Flags().StringSliceVar(&shootNetworks, "shoot-network", nil, "shoot networks to add routes for")
+	cmd.Flags().StringVar(&seedPodNetwork, "seed-network", "", "seed pod network to add mapping rules for")
```

Comment:
Please add IPv4 and maybe even double NAT so that it is more explicit what this will be used for.